tests/installation/user_settings.pm

```diff
@@ -35,23 +35,22 @@ sub run {
         return;
     }
 
-    if (check_var('SLE_PRODUCT', 'hpc')) {
-        # in case of HPC product we okay to ignore mistyped username
-        $self->enter_userinfo(max_interval => utils::VERY_SLOW_TYPING_SPEED);
-        assert_screen([qw(inst-userinfostyped-ignore-full-name inst-userinfostyped-expected-typefaces)]);
-    }
-    else {
+    if (get_var('ASSERT_BSC1122804')) {
         # retry if not typed correctly
         my $max_tries = 4;
         my $retry     = 0;
         do {
-            $self->enter_userinfo(max_interval => ($retry) ? utils::VERY_SLOW_TYPING_SPEED : undef);
+            $self->enter_userinfo(max_interval => utils::VERY_SLOW_TYPING_SPEED);
```

Comment:
I think this line should be reverted , you need to use $retry variable **here**

Reply:
Why?

The reason to have 'undef' (normal speed) is for the scenarios not affected by the bug.
Now that this if-branch is only to hit it, we want to always type slow. This will only happen to functional tests that use the setting.

Reply:
ah ok I see , make sense . please ignore my comment above than